firmware/tunerstudio/rusefi.input

```diff
@@ -1139,21 +1139,23 @@ menuDialog = main
 		subMenu = cltTimingCorrCurve,		"CLT correction", 0, {isIgnitionEnabled == 1}
 		subMenu = ignitionIatCorrTableTbl,	"IAT correction", 0, {isIgnitionEnabled == 1}
 		
-	menu = "&Cranking"
-		subMenu = crankingDialog,			"Cranking settings"
+
+		
+	menu = "&Idle and Cranking"
```

Comment:
Order in the menu title should match the order in the menu - if cranking is first in the title, it should be first in the menu